tests/gtests/graph/unit/backend/dnnl/test_rms_norm.cpp

```diff
@@ -0,0 +1,148 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <random>
+#include "gtest/gtest.h"
+
+#include "graph/unit/backend/dnnl/dnnl_test_common.hpp"
+#include "graph/unit/unit_test_common.hpp"
+#include "graph/unit/utils.hpp"
+
+namespace graph = dnnl::impl::graph;
+namespace utils = dnnl::graph::tests::unit::utils;
+
+TEST(test_rms_norm_execute, RMSNormInference) {
+    graph::engine_t *eng = get_engine();
+
+    // src shape: (2, 3, 2)
+    std::vector<float> src {1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f,
+            9.0f, 10.0f, 11.0f, 12.0f};
+    std::vector<float> scale {1.0f, 2.0f};
+    std::vector<float> dst(src.size(), 0.0f);
+
+    // Reference values computed with NumPy:
+    // RMS = sqrt(mean(x^2, axis=-1) + epsilon), output = (x / RMS) * scale
+    std::vector<float> ref_dst {0.63245427f, 2.52981707f, 0.8485278f,
+            2.26274079f, 0.90535731f, 2.17285755f, 0.93126606f, 2.12860815f,
+            0.94605894f, 2.10235321f, 0.9556189f, 2.08498669f};
```

Comment:
Correctness validation should be covered by benchdnn already? Any specific case we need to cover by gtests?

Reply:
Got it. thanks for your info. Removed the redundant acc cases. Could you please take another look?